DigitalOffice/Tests/AuthenticationServiceTests/TokenTests/CreateNewTokenTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Moq;
+using NUnit.Framework;
+using LT.DigitalOffice.AuthenticationService.Token;
+using LT.DigitalOffice.AuthenticationService.Token.Interfaces;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System.Text;
+namespace LT.DigitalOffice.AuthenticationServiceUnitTests.TokenTests
+{
+    public class CreateNewTokenTests
```

Comment:
It is necessary to change the name from TokenTests to Token.

Reply:
Fixed